tests/naive.py

```diff
@@ -190,13 +221,26 @@ def stump(T_A, m, T_B=None, exclusion_zone=None, row_wise=False, k=1):
             [distance_profile(Q, T_A, m) for Q in core.rolling_window(T_A, m)]
         )
         T_B = T_A.copy()
+        T_B_subseq_isconstant = T_A_subseq_isconstant
     else:
         ignore_trivial = False
         distance_matrix = np.array(
             [distance_profile(Q, T_B, m) for Q in core.rolling_window(T_A, m)]
         )
 
+    T_A_subseq_isconstant = rolling_isconstant(T_A, m, T_A_subseq_isconstant)
+    T_B_subseq_isconstant = rolling_isconstant(T_B, m, T_B_subseq_isconstant)
+
     distance_matrix[np.isnan(distance_matrix)] = np.inf
+    for i in range(distance_matrix.shape[0]):
+        for j in range(distance_matrix.shape[1]):
+            if np.isfinite(distance_matrix[i, j]):
+                if T_A_subseq_isconstant[i] and T_B_subseq_isconstant[j]:
+                    distance_matrix[i, j] = 0.0
+                elif T_A_subseq_isconstant[i] or T_B_subseq_isconstant[j]:
+                    distance_matrix[i, j] = np.sqrt(m)
+                else:
+                    pass
```

Comment:
It seems that `else: pass` is treated as skipped coverage in spite of the fact that the code goes through these lines. I wanted to avoid `if / elif` structure and wanted to include `else` as well. What do you think @seanlaw ?

Reply:
In this case, I accept `# pragma: no cover`